wiki/Reading_large_PDB_files.md

```diff
@@ -48,19 +48,20 @@ left for another article.
 
```

Comment:
@orbeckst is this largely superseeded by MDAnalysis? Could be useful to add a note here.

Reply:
Nowadays there are plenty of packages out there that can deal with simulation-specific big PDB-like files, MDAnalysis being one of them. I would probably remove the whole page...

Reply:
👍

Reply:
That's fine by me. Let's merge this and then do that? Maybe leave a redirection to the PDBParser page?